app/models/service_provider.rb

```diff
@@ -37,6 +37,17 @@ def ial_friendly
     end
   end
 
+  def aal_friendly
+    case default_aal
+    when 1, nil
+      ''
+    when 2, 3
+      "AAL#{default_aal}"
+    else
+      default_aal.inspect
```

Comment:
I was wondering what case would lead to a `default_aal` that wasn't `nil, 1, 2, 3`, should we have a validation?

```ruby
validates :default_aal, numericality: { only_integer: true, allow_nil: true, less_than_or_equal_to: 3 }
```

Reply:
I don't feel strongly, but if we wanted to add the validation, it would go in the `identity-validations` gem.